library/system/src/modules/FileChanges.rb

```diff
@@ -221,9 +221,61 @@ def CheckFiles(files)
       true
     end
 
+    # Check if any of the possibly new created files is really new
+    # Issue a question whether to continue if such file was manually created
+    # @param [Array<String>] files a list of files to check
+    # @return [Boolean] true if either none was changed or user agreed
+    #  to continue
+
+    def CheckNewCreatedFiles(files)
+      new_files = files - @file_checksums.keys
+
+      if new_files.size > 0
+        # Continue/Cancel question, %s is a file name
+        msg = _("File %s has been created manually.\nYaST might lose this file.")
+        if new_files.size > 1
+          # Continue/Cancel question, %s is a comma separated list of file names
+          msg = _(
+            "Files %s have been created manually.\nYaST might lose these files."
+          )
+        end
+        msg = msg % new_files.join(", ")
+        popup_file = "/filechecks_non_verbose"
+        popup_file_path = File.join(Directory.vardir, popup_file)
+        if !FileUtils.Exists(popup_file_path)
```

Comment:
This is how old (or known from before) files are handled (see CheckFiles method). This way it is consistent and there were no complaints about the behavior yet (at least AFAIK).